### PR TITLE
PR: Add alert at the top of the page, Gitter support and support for links in the copyright

### PR DESCRIPTION
**Pull request by dalthviz**
A preview:

theme

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -64,7 +64,8 @@
             "name": "Pandas Docs"
         }
     ],
-    "use_edit_page_button": True
+    "use_edit_page_button": True,
+    "alert_message": "The documentation is still a work in progress!"
```
Comment (ccordoba12) [inline docs/conf.py]:
```suggestion
    "alert_message": "This documentation is still a work in progress!"
```

**Review by ccordoba12 — COMMENTED** [review]
This looks really good, thanks @dalthviz! I only left one minor comment, otherwise I'd say it's ready to be merged.

**Comment by ccordoba12** [comment]
@juanis2112, do you agree with this change?

**Comment by juanis2112** [comment]
@ccordoba12 sounds good.

@dalthviz looks good to me!

**Review by juanis2112 — APPROVED** [review]

**Comment by dalthviz** [comment]
@ccordoba12 @juanis2112 I think this is ready to be merged

**Pull request merged**